drv/transceivers-server/src/udp.rs

```diff
@@ -398,16 +434,25 @@ impl ServerImpl {
                         0,
                     );
                 }
+
                 let mask = LogicalPortMask::from(modules);
-                let result = self.write(write, mask & !self.disabled, data);
-                ringbuf_entry!(Trace::OperationResult(result));
+                let result = if self.front_io_board_present.unwrap_or(false) {
+                    self.write(write, mask & !self.disabled, data)
+                } else {
+                    ModuleResult::new(
+                        LogicalPortMask(0),
+                        LogicalPortMask(0),
+                        mask,
+                        LogicalPortFailureTypes::default(),
+                    )
+                    .unwrap()
```

Comment:
...and this one, sorry
```suggestion
                    .unwrap_lite()
```